src/features/ansibleMetaData.ts

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable  @typescript-eslint/no-explicit-any */
+import { window, MarkdownString, ThemeColor, StatusBarItem } from "vscode";
+import { NotificationType } from "vscode-languageclient";
+import { LanguageClient } from "vscode-languageclient/node";
+import { formatAnsibleMetaData } from "./utils/formatAnsibleMetaData";
+
+/**
+ * Sends notification with active file uri as param to the server
+ * and receives notification from the server with ansible meta data associated with the opened file as param
+ *
+ * @param client Language client
+ * @param myStatusBarItem Statusbar item
+ * @param isActiveClient Boolean representing the activation status of the client
+ * @param cachedAnsibleVersion String representing the cached ansible version which is updated globally
+ *
+ * @returns String representing the cached ansible version
+ */
+export function updateAnsibleInfo(
+  client: LanguageClient,
+  myStatusBarItem: StatusBarItem,
+  isActiveClient: boolean,
+  cachedAnsibleVersion: string
+): string {
+  if (isActiveClient) {
+    myStatusBarItem.tooltip = new MarkdownString(
```

Comment:
please rename this to a cleaner name eg. `metadataStatusBar` at all applicable places

Reply:
Done. Thanks.